src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -280,7 +316,7 @@ const Apm = require("@sentry/apm"); // This is required since it patches functio
 
 Sentry.init({
     dsn: "___PUBLIC_DSN___",
-    tracesSampleRate: 0.1
+    tracesSampleRate: 1
```

Comment:
Why change it to `1`?

Reply:
Daniel, we cannot allow anybody to send 100% of their traffic to us. This burns immediately through their quota as long as the new billing is not live.

Reply:
I leave it in the responsibility of the user to set the correct value here.
1 = they see it works
0.1 = they write into support it's not working

Reply:
Okay, can we then add a comment here that they should be careful in increasing this towards 1 in prod? 1 makes total sense in dev.

Reply:
FWIW this is also inconsistent with other code examples.

Reply:
> can we then add a comment here that they should be careful in increasing this towards 1 in prod? 1 makes total sense in dev.

+1

Maybe nobody cares, but I would also find `1.0` more indicative that this is not a `0` or `1` setting, but that typical values are floats in the [0.0, 1.0] range: `0.0`, `0.1`, `0.25`...

Reply:
@HazAT 
maybe keep it at 1.0 but then put a comment before or after the code snippet that says,
"We recommend 0.1 for when you put this to production [because...]"

maybe keep it at 0.1 but then put a comment before or after the code snippet that says,
"You may want this on 1.0 if you're testing it for the first time"
"You may want this on 1.0 if you're testing it for the first time or else you might think your transactions aren't coming through"

you're 100% correct about them writing into support, as I tricked myself on this the first time I set it up, and then again a month later when I re-visisted it. So that makes me prefer to see it at 1.0 by default.